src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -408,12 +409,41 @@ namespace AppInstaller::CLI::Workflow
         context <<
             Workflow::SelectInstaller <<
             Workflow::EnsureApplicableInstaller <<
+            Workflow::ManageDependencies <<
             Workflow::InstallPackageInstaller;
     }
 
+    void ManageDependencies(Execution::Context& context)

```

Comment:
Actually as of now it will not manage any dependencies and it's only gathering dependencies info from installer. So instead of moving it to DependenciesFlow I changed its name to GetInstallerDependencies